index.html

```diff
@@ -200,14 +201,12 @@ <h1 class="review-header text-center w-100 text-light">What people say about us?
               </div>
         </section>
         <section class="wrote-about-us">
-            <h1 class="text-center w-100 text-dark">Wrote about us</h1>
-            <div class="container-fluid">
-                <div class="row d-flex justify-content-around mr-1 ml-1 line-after">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/Forbes_logo.svg.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/the-new-york-times-logo-vert.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/2000px-The_Wall_Street_Journal_Logo.svg.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/Mashable-Logo.png" alt="">
-                </div>
+            <h1 class="wrote-about-us__title">Wrote about us</h1>
+            <div class="wrote-about-us__images-container">
+                <img class="wrote-about-us__images-container__img" src="./img/Forbes_logo.svg.png" alt="">
```

Comment:
you should not create elements of elements in BEM. Please simplify this class to `wrote-about-us__img`.